cmd/rukpakctl/main.go

```diff
@@ -0,0 +1,11 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+*/
+package main
+
+import "github.com/operator-framework/rukpak/cmd/rukpakctl/cmd"
+
+func main() {
+	cmd.Execute()
```

Comment:
If we refactor the root cobra.Command into a constructor function vs. relying on a package-level variable, we can do the following:

```suggestion
	cmd := root.NewCmd()
	if err := cmd.Execute(); err != nil {
	    fmt.Fprintln(os.Stderr, err)
	    os.Exit(1)
        }
```